example.py

```diff
@@ -9,12 +9,14 @@
 # - The language's name in English
 # - The language's name in that language (its autonym)
 
+en = langcodes.get('en')
+
 for let1 in string.ascii_lowercase:
     for let2 in string.ascii_lowercase:
         for let3 in [''] + list(string.ascii_lowercase):
             code = let1 + let2 + let3
             lcode = langcodes.get(code)
             autonym = lcode.autonym()
             name = lcode.language_name()
-            if autonym != lcode.language:
+            if lcode.distance(en) < 10 or autonym != lcode.display_name(en):
```

Comment:
I've read things over, and to the extend I understand them (call it "80%"?) the PR looks good.  This change is the one place I'm stumbling, for two reasons:

1. In master, this script runs in eight seconds on my computer, and in the branch, it takes over a minute and a half.  `lcode.display_name(en)` (with `lcode = langcodes.get('af')`) takes around 1.6ms; `en.display_name(en)` takes around 1.9ms.  18,252 calls to display_name might explain the disparity, but I wanted to mention it in case
   1. performance of example.py in particular is important, or
   2. this is indicating anything important about the performance in general
2. I was running the script to see what was different between this branch and master.  Most of the changes are additions (e.g. `bxk bxk Bukusu`), which makes sense; some of them are changes to the autonym (e.g. `hbs` going from "srpski" to "srpski (latinica)"), which are obviously intended effects.  But there are also some things that disappeared, among them Afrikaans, Filipino, Hausa, Kinyarwanda, Tok Pisin, Wolof.  (Also Volapük, but, you know, whatever.)  It looks like these are languages where the English name is the same as the autonym, which makes sense given the addition of `autonym != lcode.display_name(en)`, but I wasn't sure this was the intention of that condition.